app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingActivity.kt

```diff
@@ -61,18 +67,37 @@ class AddListingActivity : AppCompatActivity() {
         galleryImageButton = findViewById(com.springsamurais.toyswap.R.id.gallery_image_button)
         itemTitleInput = findViewById(com.springsamurais.toyswap.R.id.item_title_input)
         itemDescriptionInput = findViewById(com.springsamurais.toyswap.R.id.item_description_input)
-        conditionSpinner = findViewById(com.springsamurais.toyswap.R.id.condition_spinner)
-        categorySpinner = findViewById(com.springsamurais.toyswap.R.id.category_spinner)
+
```

Comment:
If we get time (haha) it would be nice to refactor this with data-binding and click handlers, but this is an unnecessary stretch goal at the moment.